cli/actions/__tests__/build.test.js

```diff
@@ -0,0 +1,74 @@
+const shell = {
```

Comment:
There is a global shell mock, but build needed a slightly different behavior, so it provides its own here.